spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject(:user) do
+    described_class.new(username: 'wilson', email: 'wilson@gmail.com', password: 'password', rating: 2000,
```

Comment:
In my opinion it would be worth it to implement factories (uniform database object builders with random field inputs) for testing specs. In the past we've done this with [FFaker](https://github.com/ffaker/ffaker) and [FactoryBot](https://github.com/thoughtbot/factory_bot_rails). @ConnorJC3 has done this before so he may be able to help.

Reply:
This is a good idea, currently looking into improving spec testing with FFaker and FactoryBot 👍